Dockerfile

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 
 # Build the manager binary
-FROM golang:1.20.2 as builder
```

Comment:
lets update the dockerfile in https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/main/hack/ccm/Dockerfile as well

Reply:
my bad, I see the one as well here - https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/pull/1186